dev/com.ibm.ws.junit.extensions/bnd.bnd

```diff
@@ -0,0 +1,25 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+bVersion=1.0
+
```

Comment:
In this file you'll want to either set `test.project: true` or `publish.wlp.jar.disabled: true`, otherwise a bundle will get generated for this project and end up in wlp/lib/

Reply:
I updated it to use test.project: true

Reply:
Turns out using test.project: true adds extra dependencies which re-introduce cycles. so now I use both:

publish.wlp.jar.disabled: true
instrument.disabled: true